walkthroughs/eks-getting-started/yelb-appserver-v2/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM public.ecr.aws/bitnami/ruby:latest
 MAINTAINER massimo@it20.info
 
+## Oct 10 2022: 
+## Changed all the flags from --no-ri --no-rdoc  to --no-document 
+## install zlib1g and zlib1g-dev packages 
+## install puma
+
```

Comment:
Overall changes look good to me. I'd prefer not including Changelog in the Dockerfile.

Reply:
Have removed the Changelog in the docker file.